Doc/c-api/arg.rst

```diff
@@ -652,9 +652,9 @@ Building values
 
    ``O&`` (object) [*converter*, *anything*]
       Convert *anything* to a Python object through a *converter* function.  The
-      function is called with *anything* (which should be compatible with :c:type:`void
-      \*`) as its argument and should return a "new" Python object, or ``NULL`` if an
-      error occurred.
+	  function is called with *anything* (which should be compatible with
+	  :c:expr:`void*`)as its argument and should return a "new" Python object, or
+	  ``NULL`` if an error occurred.
```

Comment:
Seems the tab character is used here. Please replace all tab characters with spaces and check that lines are correctly aligned.